subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,85 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540_gpio
+DT_COMPAT_SKYWORKS_SKY66112_11 := skyworks,sky66112-11
+
+config MPSL_FEM
```

Comment:
Is it supported on nrf53 already? If no, we should prevent the user from selecting this configuration

Reply:
I added `depends on !(SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET)`

Reply:
This is temporary, right?

Reply:
Yes, support for FEMs for nRF53 is on our roadmap, so from this point of view this `depends on !SOC_NRF53 ..` is temporary